$MirOS: ports/net/rsync/patches/patch-rsyncd_conf_5,v 1.3 2008/12/30 03:01:32 tg Exp $
--- rsyncd.conf.5.orig	Fri Sep 23 16:35:31 2011
+++ rsyncd.conf.5	Wed Jan  4 23:02:19 2012
@@ -764,8 +764,8 @@ A more sophisticated example would be:
 .PP 
 .nf 
 
-uid = nobody
-gid = nobody
+uid = _rsync
+gid = _rsync
 use chroot = yes
 max connections = 4
 syslog facility = local5
